Multisig/UI/App/UIKit/Transaction/TransactionListViewController/Cells/TransactionsListConflictHeaderTableViewCell.swift

```diff
@@ -26,6 +26,15 @@ class TransactionsListConflictHeaderTableViewCell: UITableViewCell {
     }
 
     @IBAction func learnMoreButtonTouched(_ sender: Any) {
-        UIApplication.shared.sendAction(#selector(UIViewController.didTapExternalURLCell(_:)), to: nil, from: self, for: nil)
+        UIApplication.shared.sendAction(#selector(UIViewController.didTapTransactionsListConflictHeaderTableViewCell(_:)), to: nil, from: self, for: nil)
```

Comment:
hm... instead of this functionality duplication, why don't we create a protocol "ExternalURLSource" that has the "url" getter variable, and make the didTapExternalURLCell's argument Any. Inside that function, then, we can if-let cast the sender to the ExternalURLSource and obtain the url.

That way it would be generic and extendable.

Reply:
Honestly I thought to make it this way first :D, but I changed my mind to minimize the changes, but I will change right awat